.docksal/commands/config-capture

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+## Capture Configuration from a any environment.
+##
+## Usage: fin config-capture [environment id]
+## Defaults to the primary environment but can take an environment name as a parameter: fin capture-config pr-123
+
+# Abort if anything fails
+set -e
+
+ENVIRONMENT=${1:-"$hostingenv"}
+
+# Sets todays datetime.
+var_date=$(date +"%Y-%m-%d-%H-%M-%S")
+
+# Install the terminus-rsync-plugin plugin.
+echo -e "Install the terminus-rsync-plugin plugin"
```

Comment:
What happens if the plugin is already installed? Does it try to install it again?

Reply:
`[notice] pantheon-systems/terminus-rsync-plugin is already installed.`

Reply:
It works fin as is.

The only way I could see figuring out if it is installed or not was:

```
❯ fin terminus self:plugin:list
 ----------------------- ------------------------------------------------------------ ------------------- ---------------- -----------------
  Name                    Description                                                  Installed Version   Latest Version   Compatible With
 ----------------------- ------------------------------------------------------------ ------------------- ---------------- -----------------
  terminus-rsync-plugin   Rsync - Terminus Plugin that provides a quick shortcut for   1.1.0               1.1.0            ^1|^2
                          rsync-ing files to and from a Pantheon site.
 ----------------------- ------------------------------------------------------------ ------------------- ---------------- -----------------
 ```

But I can't see an easy way to be able to parse that into an IF statement.

This is similar to the `drupal-starter-installer` project with the `gh` plugins.  If they are installed, trying to install them again seems just as fast as checking if they are installed.

Reply:
Updated.